### PR TITLE
Remove blockHash arg from pauseDeposits() method

### PR DESCRIPTION
**Pull request by Psirex**

### CODE REVIEW

File: contracts/0.8.9/DepositSecurityModule.sol [hunk 1]
```diff
@@ -326,23 +326,17 @@ contract DepositSecurityModule {
...
         address guardianAddr = msg.sender;
         int256 guardianIndex = _getGuardianIndex(msg.sender);
 
         if (guardianIndex == -1) {
-            bytes32 msgHash = keccak256(abi.encodePacked(
-                PAUSE_MESSAGE_PREFIX,
-                blockNumber,
-                blockHash
-            ));
+            bytes32 msgHash = keccak256(abi.encodePacked(PAUSE_MESSAGE_PREFIX,blockNumber));
```
Comment (ujenjt) [inline contracts/0.8.9/DepositSecurityModule.sol:339]: PAUSE_MESSAGE_PREFIX,blockNumber -> PAUSE_MESSAGE_PREFIX, blockNumber

Comment (ujenjt) [reply contracts/0.8.9/DepositSecurityModule.sol:339]: space plz

**Review by ujenjt — CHANGES_REQUESTED** [review]
Style fix, the rest looks good

File: contracts/0.8.9/DepositSecurityModule.sol (hunk 1 above)
Comment (Psirex) [reply contracts/0.8.9/DepositSecurityModule.sol]: Done

**Pull request merged**